wandb/sdk/artifacts/artifact.py

```diff
@@ -1414,13 +1429,10 @@ def add(self, obj: WBValue, name: StrPath) -> ArtifactManifestEntry:
             data_types.Object3D,
             data_types.Molecule,
             data_types._SavedModel,
-        ]
-
-        if not any(isinstance(obj, t) for t in allowed_types):
+        )
+        if not isinstance(obj, allowed_types):
```

Comment:
note: isinstance can take a tuple as the second arg, and in most cases this is marginally if negligibly more performant